tjmonopix2/analysis/plotting.py

```diff
@@ -550,16 +567,16 @@ def _add_text(self, fig):
         if self.internal:
             fig.text(0.1, 1, 'Internal', fontsize=16, color='r', rotation=45, bbox=dict(boxstyle='round', facecolor='white', edgecolor='red', alpha=0.7), transform=fig.transFigure)
 
-    def _convert_to_e(self, dac, use_offset=True):
+    def _convert_to_e(self, dac, use_offset=False):
         if use_offset:
             e = dac * self.calibration['e_conversion_slope'] + self.calibration['e_conversion_offset']
             de = math.sqrt((dac * self.calibration['e_conversion_slope_error'])**2 + self.calibration['e_conversion_offset_error']**2)
         else:
-            e = dac * self.calibration['e_conversion_slope']
-            de = dac * self.calibration['e_conversion_slope_error']
+            e = dac * self.electron_conversion
+            de = dac * self.electron_conversion * 0.15  # TODO: Replace generic 15% error of conversion factor
```

Comment:
"Replace" might not be the best term here. For now, we assume 15% error but have to estimate more precisely in the future.